src/sentry/api/serializers/models/sentry_app_component.py

```diff
@@ -13,5 +13,8 @@ def serialize(self, obj, attrs, user):
             'uuid': six.binary_type(obj.uuid),
             'type': obj.type,
             'schema': obj.schema,
-            'sentryAppId': obj.sentry_app_id,
```

Comment:
Do folks ever need to go from a component response back to the app? Having the ID lets you cook up URLs back to the sentry app.

Reply:
Yea, it's probably worth having ID in the embedded object.